packages/http-client-csharp/eng/scripts/Test-CadlRanch.ps1

```diff
@@ -87,7 +86,14 @@ foreach ($directory in $directories) {
     }
 
     Write-Host "Testing $subPath" -ForegroundColor Cyan
-    $command  = "dotnet test $cadlRanchCsproj --filter `"FullyQualifiedName~$testFilter`""
```

Comment:
Is $testFilter used anymore?

Reply:
no, we do not need that local variable any more.